src/fabric/api/fixed-asset/node/lib/fixed-asset.js

```diff
@@ -110,6 +140,40 @@ class Asset extends Contract {
         }
     }
 
+    /**
+     * Create a set of Assets in the registry based on the body that is provided of the form
+     * [{
+     *   uuid: unique identifier
+     *   creator: the creator
+     *   bytesize: target bytesize of asset
+     *   content: variable content
+     * }, ...]
+     * @param {Context} ctx the context
+     * @param {String} batch the content to persist within an array
+     */
+    async createPrivateAssetsFromBatch(ctx, batch_size) {
+
+        if (isVerbose) {
+            console.log('Entering createPrivateAssetsFromBatch');
+        }
+
+        const transientContent = ctx.stub.getTransient().get('content');
+        const transientData = JSON.parse(transientContent);
+
+        for(let i=0; i<batch_size; i++) {
```

Comment:
`for (`  (add the space for consistency, it's petty - but you did it elsewhere!)